api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -45,12 +45,12 @@ function validatorReducer(network, validator) {
     networkId: network.id,
     chainId: network.chain_id,
     operatorAddress: validator.accountId,
-    website:
-      validator.identity.web && validator.identity.web !== ``
-        ? validator.identity.web
-        : ``,
+    website: validator.identity.web ? validator.identity.web : ``,
```

Comment:
do we need to check for `validator.identity`?

Reply:
no, we don't, since `api.derive.accounts.info()` always returns an Object, even when the fields are empty. I have checked